app/assets/stylesheets/application.scss

```diff
@@ -19,12 +19,22 @@
 //paletee of colors #C63D0F #3B3738 #fdf3e7 #7E8F7C //
 
 //General
-.login-in {
-  padding-top: 8%;
-}
 
 body {
   background-color: #fdf3e7;
+  font-family: 'Amaranth', sans-serif;
+}
+
+h1 {
+  font-family: 'Amaranth', sans-serif;
+}
+
+.panel{
+  background-color: #fafcfd;
```

Comment:
Color literals like `#fafcfd` should only be used in variable declarations; they should be referred to via variable everywhere else.